lib/view/page/idea_details_page.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class IdeaDetailsPage extends StatelessWidget {
+  final String title;
+  final String description;
```

Comment:
Idea details page should take an Idea as an argument instead of title & description.